README.md

```diff
@@ -1,3 +1,48 @@
-# repo-template
+# Jira Plugin for Justification Verification Service (JVS)
 
-**This is not an official Google product.**
+**Jira plugin for JVS is not an official Google product.**
+
+This repository contains components related to the plugin for justification 
+verification service.
+
+## Installation
+
+You can use the provided Terraform module to setup the basic infrastructure
+needed for this service. Otherwise you can refer to the provided module to see
+how to build your own Terraform from scratch.
+
+```terraform
+module "jvs" {
+  source     = "git::https://github.com/abcxyz/jvs.git//terraform/e2e?ref=main" # this should be pinned to the SHA desired
+  project_id = "YOUR_PROJECT_ID"
+
+  # Specify who can access JVS.
+  jvs_invoker_members = ["user:foo@example.com"]
+
+  # Use your own domain.
+  jvs_api_domain = "api.jvs.example.com"
+  jvs_ui_domain  = "web.jvs.example.com"
+
+  iap_support_email = "support@example.com"
+
+  jvs_container_image = "us-docker.pkg.dev/abcxyz-artifacts/docker-images/jvsctl:0.0.5-amd64"
+
+
+  # Specify the Email address where alert notifications send to.
+  notification_channel_email = "foo@example.com"
+
+  jvs_prober_image = "us-docker.pkg.dev/abcxyz-artifacts/docker-images/jvs-prober:0.0.5-amd64"
+
+  # The cloud run url for jvs api service or app address.
+  prober_audience = "https://example-api-jvs-cloud-run.run.app"
```

Comment:
This feels like a circular dependency, as it is the cloud run created in the same module? @sailorlqh @yolocs can it be fixed by using the cloud run url generated directly in the module?

Reply:
I think you are right if we use the cloud run service's url. But the audience not necessarily be the cloud run service url, it can also be the gcloud app id. We made this an variable here because we are using the gcloud app id (https://github.com/abcxyz/jvs/blob/main/docs/prober-and-monitoring.md?plain=1#L45)

Reply:
I forgot why we use gcloud app id, was it a temporary workaround, should it be cloud run url?

Reply:
It's not a work around, we thought its a better method as the gcloud app id is a universal one, it more robust than service url since service url can change.

Reply:
Discussed with Qinhang offline, it is probably easier and consistent to hard code the cloud run url in the module. @yolocs What do you think?

Reply:
For now, add a comment to use the gcloud app id is fine.

Reply:
Done